components/brave_new_tab_ui/components/default/rewards/index.tsx

```diff
@@ -226,23 +251,36 @@ class Rewards extends React.PureComponent<RewardsProps, {}> {
   }
 
   renderNotifications = () => {
-    const { promotions, onDismissNotification } = this.props
+    let { promotions, onDismissNotification } = this.props
 
-    if (!promotions) {
-      return null
-    }
+    // const showDummyPromotion = true
```

Comment:
I left this comment in intentionally, to help the next person be able to get something to show in the rewards notifications since it's very difficult to get real data to show up here.